app/src/main/java/edu/umd/cmsc436/mstestsuite/MainActivity.java

```diff
@@ -185,6 +185,24 @@ public void run() {
         });
     }
 
+    public void sendFeedbackToDoc() {
+        setContentView(R.layout.feedback);
+        Button mButton = (Button) findViewById(R.id.send_email_btn);
+        final EditText mEditText = (EditText) findViewById(R.id.feedback_edit_text);
+
+        mButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent i = new Intent(Intent.ACTION_SEND);
+                i.putExtra(Intent.EXTRA_EMAIL, new String[]{"lisamaszkiewicz@gmail.com"});
```

Comment:
We don't need to have any email there right now, just launch the email intent with a body